packages/vue-i18n-core/src/warnings.ts

```diff
@@ -14,7 +18,11 @@ export const warnMessages: { [code: number]: string } = {
   [I18nWarnCodes.FALLBACK_TO_ROOT]: `Fall back to {type} '{key}' with root locale.`,
   [I18nWarnCodes.NOT_FOUND_PARENT_SCOPE]: `Not found parent scope. use the global scope.`,
   [I18nWarnCodes.IGNORE_OBJ_FLATTEN]: `Ignore object flatten: '{key}' key has an string value`,
-  [I18nWarnCodes.DEPRECATE_TC]: `'tc' and '$tc' has been deprecated in v10. Use 't' or '$t' instead. 'tc' and '$tc’ are going to remove in v11.`
+  [I18nWarnCodes.DEPRECATE_TC]: `'tc' and '$tc' has been deprecated in v10. Use 't' or '$t' instead. 'tc' and '$tc’ are going to remove in v11.`,
```

Comment:
I suppose it's not necessarily related this PR as it's not specific to legacy API, but we should make sure to remove `tc` and `$tc` for v11 as well. I assume this will happen in a separate PR?

Reply:
Yeah, I'm going to open a separate PR.
When I will merge this PR, I will release v11 beta